airbyte-integrations/connectors/destination-motherduck/destination_motherduck/processors/duckdb.py

```diff
@@ -261,17 +261,10 @@ def _write_temp_table_to_target_table(
         sync_mode: DestinationSyncMode,
     ) -> None:
         """Write the temp table into the final table using the provided write strategy."""
-        if sync_mode == DestinationSyncMode.overwrite:
-            # Note: No need to check for schema compatibility
-            # here, because we are fully replacing the table.
-            self._swap_temp_table_with_final_table(
```

Comment:
I don't see this method used anywhere else, so we can probably deprecate it.

Reply:
The method is defined in the parent class so I won't touch it for now as it can be important to other destinations.